packages/plugin-sass/src/index.ts

```diff
@@ -206,106 +201,157 @@ ${cbClose}`
   };
 }
 
-/** transform color */
-export function transformColor(value: ParsedColorToken['$value']): string {
-  return String(value);
```

Comment:
Slightly-irrelevant to this PR, but also realized with the simplification in #114, we can now use the same transformers for both plugin-css and plugin-sass, reducing the overall bug surface area.

As far as “transformers” are concerned in plugin-css and plugin-sass, these **ONLY** deal with raw, normalized values. Resolving aliases is now done **in the plugin core**. Again, slow realization that lead to some cleanup in how plugins are utilized that I’m happy with.

This isn’t really a fundamental API change or anything, but as others build plugins, figuring this stuff out and setting patterns makes it so others don’t have to wrestle with as much.